lib/trento_web/views/v1/sap_system_view.ex

```diff
@@ -125,6 +125,12 @@ defmodule TrentoWeb.V1.SapSystemView do
 
   def render("sap_system_deregistered.json", %{id: id, sid: sid}), do: %{id: id, sid: sid}
 
+  def render("database_deregistered.json", %{
```

Comment:
This could be one liner, right? It could even be `def render("database_deregistered.json", database), do: database`.
But it is alright to leave as the one above